packages/schema/src/config/build.ts

```diff
@@ -38,6 +39,23 @@ export default defineUntypedSchema({
     },
   },
 
+  /**
+   * Log level when building logs.
+   *
+   * Defaults to 'silent' when running in CI or when a TTY is not available.
+   * This option is then used as 'silent' in Vite and 'none' in Webpack
+   *
+   * @type {'silent' | 'info' | 'verbose'}
+   */
+  logLevel: {
+    $resolve: (val) => {
+      if (val && !['silent', 'info', 'verbose'].includes(val)) {
+        console.error('Invalid `logLevel` option. Must be one of: `silent`, `info`, `verbose`.')
```

Comment:
(we could also include `val` in message for better DX)

```suggestion
        console.warn('Invalid `logLevel` option. Must be one of: `silent`, `info`, `verbose`.')
```